TestProject/TestProject/Entities/ImplementersPlugin/ProtocolEntity.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  ProtocolsEntity.swift
+//  TestProject
+//
+//  Created by Muhammad Taifoor Saleem on 27/01/2022.
+//
+
+import Foundation
+
+//Refer to To-Do List DB Requisitions
+
+protocol ProtocolEntity
```

Comment:
We can rename this to DataSource Protocol. Reference image attached.
<img width="502" alt="Screenshot 2022-02-09 at 1 36 45 PM" src="https://user-images.githubusercontent.com/72196767/153157419-30cd9afd-19d5-4df3-8608-a034e0daf03d.png">

Reply:
Done